code/python/pipelines-workshop/requirements.txt

```diff
@@ -1,11 +1 @@
-aws-cdk.core
-aws-cdk.aws_lambda
-aws-cdk.aws-apigateway
-aws-cdk.pipelines
-aws-cdk.aws_dynamodb
-aws-cdk.aws_codecommit
-aws-cdk.aws_codebuild
-aws-cdk.aws_codepipeline
-aws-cdk.aws_codepipeline_actions
-cdk_dynamo_table_viewer
```

Comment:
the `main-workshop` takes advantage of `setup.py` instead of using requirements.txt so for sake of consistency I replicated that configuration.